release/models/ethernet-segments/openconfig-ethernet-segments.yang

```diff
@@ -199,6 +206,15 @@ grouping evpn-ethernet-segment-df-config {
         "RFC 7432: BGP MPLS-Based Ethernet VPN";
     }
 
+    leaf es-import-route-target {
+      type oc-yang-types:mac-address;
+      description
+        "Received ES routes with an ES-Import extended community set to this value
+        are imported into this ethernet segment";
```

Comment:
Can you add text regarding auto derived and what happens if configured?  ie:

"When configured by a user, the es-import-route-target will be used, otherwise the system automatically derives the value used."
Also, end the sentence with a period please.  Thanks!

Reply:
Setting to last call for comments.  THis will merge Oct 21, 2024

Reply:
Done